### ISSUE TITLE
Use ./play.it installation scripts

### ISSUE THREAD

**Issue by tatapoum** [lutris/lutris#282]
The [play.it project](http://wiki.dotslashplay.it/en/start) automatizes the installation of linux games with bash scripts.
By now the list is about one hundred games and it is regurlarly growing.

Would it be possible to reuse these scripts within lutris ?

**Comment by strycore** [comment]
The whole play.it project is very Debian specific and there's a lot of duplicate functionality between these and what Lutris does, so using those scripts directly won't be an option.

What I can do is write some scripts to make sure that we have installers for everything proposed by play.it.

**Comment by vv221** [comment]
Hello World!
I’m the main developper of ./play.it, and I’m at your disposal if you want some help interpreting the data you can find in ./play.it scripts so you can extract what is interesting in them for Lutris.
By the way, there are already a couple Archlinux ./play.it scripts that seem to work quite well, and an official support of Archlinux (and derivatives) by ./play.it should come at some point. RPM support is planned too, actually the "Debian-centrism" of the project is only due to it being a "pet project" so I needed to keep my objectives modest. Now that everything is working as intended on Debian and derivatives, support for multiple distributions is my top priority ;)

TL;DR:
If you need help supporting a game that already has a ./play.it script, get in touch with me and I’ll help you. My e-mail can be found in the scripts headers.
